rclcpp/include/rclcpp/macros.hpp

```diff
@@ -108,7 +108,4 @@
   __RCLCPP_UNIQUE_PTR_ALIAS(__VA_ARGS__) \
   __RCLCPP_MAKE_UNIQUE_DEFINITION(__VA_ARGS__)
 
-#define RCLCPP_STRING_JOIN(arg1, arg2) RCLCPP_DO_STRING_JOIN(arg1, arg2)
-#define RCLCPP_DO_STRING_JOIN(arg1, arg2) arg1 ## arg2
```

Comment:
I'm not sure it's possible but deprecating these first would be nice. I don't know if you can get a conditional `#warning` into this, probably not. If not, maybe we should just leave it? I don't like having dead code lying around, but this is trivial and we can't know if someone is using this as it is currently part of the public API. It would also prevent us from backporting this at all (maybe we don't care about that).

Reply:
reverted in 92da89e03244c2a99cf3d18748533e26d3408800